psalm.xml

```diff
@@ -13,6 +13,7 @@
        inferPropertyTypesFromConstructor="false"
        findUnusedPsalmSuppress="true"
        cacheDirectory="/tmp/psalm"
+       allowStringToStandInForClass="true"
```

Comment:
Why do you need this ? We're not supposed to

Reply:
Psalm was complaining and it fixed my issue at some point but now it's been refactored I don't need it anymore. It's now removed.